winrt/lib/text/CanvasFontFace.abi.idl

```diff
@@ -82,6 +82,19 @@ namespace Microsoft.Graphics.Canvas.Text
         float AscenderOffset;
     } CanvasGlyph;
 
+    [version(VERSION)]
+    typedef struct CanvasGlyphMetrics
+    {
+        float LeftSideBearing;
+        float AdvanceWidth;
+        float RightSideBearing;
+        float TopSideBearing;
+        float AdvanceHeight;
+        float BottomSideBearing;
+        float VerticalOriginY;
```

Comment:
I think this should be called just VerticalOrigin.   The "Y" part is implied by it being Vertical.